src/analysis/normed_space/conformal.lean

```diff
@@ -0,0 +1,249 @@
+/-
+Copyright (c) 2021 Yourong Zang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yourong Zang
+-/
+import geometry.euclidean.basic
+import geometry.manifold.charted_space
```

Comment:
This file no longer needs to import the manifolds library.

Also, on a structural level, I'd propose putting the "linear algebra" lemmas about `is_conformal_map` in a different file from the "calculus" lemmas about `conformal[_on]/[_at]`.

- "linear algebra" lemmas -- keep in this file or move to the end of `analysis.normed_space.inner_product`
- "calculus" lemmas -- move to a new file in the `analysis.calculus` folder